superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -292,15 +289,18 @@ function QueryList({ addDangerToast, addSuccessToast }: QueryListProps) {
       {
         accessor: 'sql',
         Header: t('SQL'),
-        Cell: ({
-          row: {
-            original: { sql },
-          },
-        }: any) => {
+        Cell: ({ row: { original, id } }: any) => {
           return (
-            <StyledSyntaxHighlighter language="sql" style={github}>
-              {shortenSQL(sql)}
-            </StyledSyntaxHighlighter>
+            <div
+              tabIndex={0}
+              role="button"
+              data-test={`open-sql-preview-${id}`}
+              onClick={() => setQueryCurrentlyPreviewing(original)}
+            >
+              <StyledSyntaxHighlighter language="sql" style={github}>
```

Comment:
What's the difference between setting the `language` prop here and calling `SyntaxHighlighter.registerLanguage` above? Does the latter just load in the highlighting logic for while former explicitly sets it?

Reply:
I think the SyntaxHighlighter ships with a few languages, and probably an interface to create your own. I think the `registerLanguage` step is to slim down the component and not include a bunch of unnecessary code that won't be used. I'll scan the codebase for other languages we use with SyntaxHighlighter and include those too.